README.md

```diff
@@ -7,8 +13,16 @@ This is the primary repo for Space Station 14. To prevent people forking RobustT
 If you want to host or create content for SS14, this is the repo you need. It contains both RobustToolbox and the content pack for development of new content packs.
 
 ## Links
+<div class="header" align="center">
 
-[Website](https://spacestation14.io/) | [Discord](https://discord.ss14.io/) | [Forum](https://forum.spacestation14.io/) | [Steam](https://store.steampowered.com/app/1255460/Space_Station_14/) | [Standalone Download](https://spacestation14.io/about/nightlies/)
+[![Website](https://img.shields.io/badge/Website-grey?style=for-the-badge&logo=homepage&logoColor=white)](https://spacestation14.io/)
+[![Discord](https://img.shields.io/discord/310555209753690112?style=for-the-badge&logo=Discord&logoColor=white&label=Discord)](https://discord.ss14.io/)
+[![Forum](https://img.shields.io/badge/Forum-grey?style=for-the-badge&logo=formspree&logoColor=white)](https://forum.spacestation14.io/)
+[![Steam](https://img.shields.io/badge/Steam-Playtest-g?style=for-the-badge&logo=steam&logoColor=white)](https://store.steampowered.com/app/1255460/Space_Station_14/)
+[![Standalone Download](https://img.shields.io/badge/Standalone_Download-grey?style=for-the-badge&logo=googlecloudstorage&logoColor=white)](https://spacestation14.io/about/nightlies/)
+[![Standalone Download](https://img.shields.io/badge/Space_Wizards_Development_Wiki-grey?style=for-the-badge&logo=gitbook&logoColor=white)](https://docs.spacestation14.com/)
```

Comment:
Change all io (exept discord) to com

Although I don't really like how much space they take up now. I also have concerns for readability (dyslexia fonts are unable to be applied on these shields by browser extensions)